gce/notebook/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM jupyter/base-notebook
+
+USER root
+RUN apt-get update \
+  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+USER $NB_USER
+
+RUN conda install --yes -c conda-forge \
+    dask \
+    distributed \
+    numpy \
+    pandas \
+    numba \
+    nomkl \
+    fastparquet \
+    zict \
+    blosc \
+    cytoolz \
+    ipywidgets \
+    git \
+    jupyterlab \
+    xarray \
+    netcdf4 \
+    scipy \
+    && conda clean -tipsy
+
+RUN pip install git+https://github.com/jupyterhub/nbserverproxy
+RUN jupyter serverextension enable --py nbserverproxy
+RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager@0.31
+
+RUN pip install click jedi kubernetes --upgrade
+
+RUN pip install git+https://github.com/yuvipanda/daskernetes \
+                git+https://github.com/zarr-developers/zarr \
+                git+https://github.com/jhamman/xarray@fix/zarr_set_attrs \
+                git+https://github.com/martindurant/gcsfs.git@fuse \
+                fusepy
+
+USER root
+RUN mkdir /gcs && chown -R $NB_USER /gcs
+COPY prepare.sh /usr/bin/prepare.sh
+RUN chmod +x /usr/bin/prepare.sh
+RUN mkdir /home/$NB_USER/examples && chown -R $NB_USER /home/$NB_USER/examples
+COPY examples/ /home/$NB_USER/examples
```

Comment:
@yuvipanda will this work or will the user's home directory be overridden by jupyterhub?